From ae6f06d34dbb91cc454189f43972538ea9eb7968 Mon Sep 17 00:00:00 2001
From: Colin <colin@uninsane.org>
Date: Thu, 29 Jan 2026 01:30:34 +0000
Subject: [PATCH] gst_all_1.gst-plugins-rs: passthru `selectedPlugins`

this allows one to more easily blacklist specific plugins, e.g.

```
gst_all_1.gst-plugins-rs.override {
  plugins = lib.remove "ffv1" gst_all_1.gst-plugins-rs.selectedPlugins;
}
```
---
 pkgs/development/libraries/gstreamer/rs/default.nix | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pkgs/development/libraries/gstreamer/rs/default.nix b/pkgs/development/libraries/gstreamer/rs/default.nix
index 01beb25328864c9e27d369fd3dfcea1d25a34d48..2cae57eacab76a7144947df08c93ee722fa505b4 100644
--- a/pkgs/development/libraries/gstreamer/rs/default.nix
+++ b/pkgs/development/libraries/gstreamer/rs/default.nix
@@ -335,6 +335,7 @@ stdenv.mkDerivation (finalAttrs: {
   '';
 
   passthru = {
+    inherit selectedPlugins;
     tests.pkg-config = testers.hasPkgConfigModules {
       package = finalAttrs.finalPackage;
       versionCheck = true;
